FirebaseStorageSwift/Tests/Unit/StorageTests.swift

```diff
@@ -23,8 +23,16 @@ import SharedTestUtilities
 
 import XCTest
 
-@MainActor
 class StorageTests: XCTestCase {
+  static var app: FirebaseApp?
+  override class func setUp() {
+    let options = FirebaseOptions(googleAppID: "0:0000000000000:ios:0000000000000000",
+                                  gcmSenderID: "00000000000000000-00000000000-000000000")
+    options.projectID = "myProjectID"
+    FirebaseApp.configure(name: "test", options: options)
```

Comment:
Same issue as before in the component tests - this calls `configure` but doesn't need to.

Reply:
Without it, there's this crash:
![Screen Shot 2022-03-30 at 7 01 20 AM](https://user-images.githubusercontent.com/73870/160852896-8b79f3ac-2c76-47d9-bc4b-de5c965b707d.png)

Reply:
Discussed offline, line below needs to be adjusted.